image-layout.md

```diff
@@ -199,7 +200,11 @@ Those tags will often be represented in an image-layout repository with matching
 }
 ```
 
-This illustrates an index that provides two named manifest references and an auxiliary mediatype for this image layout.
+This illustrates an index that provides two named references and an auxiliary mediatype for this image layout.
 
+The first named reference (`"stable-release"`) points to another index that might contain multiple references with several platform specification and annotations.
+Note that the `org.opencontainers.image.ref.name` annotation SHOULD only be considered valid when on descriptors on `index.json`.
```

Comment:
Maybe link to `annotations.md` for this so folks can see that we're referencing a requirement defined there.